openedx/adg/lms/applications/constants.py

```diff
@@ -118,3 +118,17 @@
 
 PROGRAM_PREREQ_COURSES_TITLE = f'<strong>{_("Program Prerequisite Courses:")}</strong>'
 BU_PREREQ_COURSES_TITLE = f'<strong>{_("Business Unit Prerequisite Courses:")}</strong>'
+
+
+APPLICATION_WAITLISTED_CONGRATS = _(
+    "Hey {username}, Thank you for visiting again. Your application is still under review."
+)
+APPLICATION_WAITLISTED_INSTRUCTION = f'{_("Admin has waitlisted your application. Please come back and ")}' \
+                                     f'{_("revisit again in sometime to check your application status.")}'
```

Comment:
Pls use parenthesis and avoid backslashes as preferred by pep8. Also here we have broken down the sentence into 2 sentences for translation. It will sent in 2 piece to transifex and translators won't be able to understand the complete context. Let me know if there is any ambiguity/